pkg/operator/controller/externaldns/deployment_test.go

```diff
@@ -30,77 +30,780 @@ import (
 )
 
 const (
-	namespace = "externaldns"
-	name      = "test"
-	image     = "bitname/external-dns:latest"
+	namespace   = "externaldns"
+	name        = "test"
+	image       = "bitname/external-dns:latest"
```

Comment:
It doesn't really matter for these tests as we are not running an instance of ExternalDNS. It could be any name. We just check the logic of the functions which are supposed to ensure the deployment. I think that's why Steve made this `bitname` (not `bitnami`) joke :)